pages/onboarding_project.ipynb

```diff
@@ -2,197 +2,206 @@
     "2. Click on a fork button.\n",
-    "3. When a fork is created click on the code button and copy an adress of your forked repository.\n",
-    "4. Run following command using copied adress:"
-   ],
-   "metadata": {}
+    "3. When a fork is created click on the code button and copy an address of your forked repository.\n",
+    "4. Run following command using copied address:"
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": 43,
-   "source": [
-    "!git clone https://github.com/$JUPYTERHUB_USER/apps.git\n",
-    "%cd apps"
-   ],
+   "execution_count": 3,
+   "metadata": {},
    "outputs": [
     {
-     "output_type": "stream",
      "name": "stdout",
+     "output_type": "stream",
      "text": [
       "Cloning into 'apps'...\n",
-      "remote: Enumerating objects: 9615, done.\u001b[K\n",
-      "remote: Counting objects: 100% (3915/3915), done.\u001b[K\n",
-      "remote: Compressing objects: 100% (1847/1847), done.\u001b[K\n",
-      "remote: Total 9615 (delta 2166), reused 3341 (delta 1884), pack-reused 5700\u001b[K\n",
-      "Receiving objects: 100% (9615/9615), 2.19 MiB | 2.37 MiB/s, done.\n",
-      "Resolving deltas: 100% (4650/4650), done.\n",
-      "/home/mdrla/work/operate-first/hitchhikers-guide/apps\n"
+      "remote: Enumerating objects: 12377, done.\u001b[K\n",
+      "remote: Counting objects: 100% (1658/1658), done.\u001b[K\n",
+      "remote: Compressing objects: 100% (877/877), done.\u001b[K\n",
+      "remote: Total 12377 (delta 743), reused 1561 (delta 686), pack-reused 10719\u001b[K\n",
+      "Receiving objects: 100% (12377/12377), 2.63 MiB | 4.05 MiB/s, done.\n",
+      "Resolving deltas: 100% (6003/6003), done.\n",
+      "/home/anand/Documents/4n4nd/hitchhikers-guide/pages/apps\n"
      ]
     }
    ],
-   "metadata": {}
-  },
-  {
-   "cell_type": "markdown",
-   "source": [
-    "#### Define important variables\n",
-    "Define variables which are used later in the tutorial."
-   ],
-   "metadata": {}
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 44,
    "source": [
-    "NAMESPACE_NAME=\"my-namespace\"\n",
-    "NAMESPACE_DISPLAY_NAME=\"My-namespace\"\n",
-    "TEAM_NAME=\"awesome-team\"\n",
-    "TARGET_CLUSTER=\"moc/zero\""
-   ],
-   "outputs": [],
-   "metadata": {}
+    "!git clone https://github.com/$GITHUB_USERNAME/apps.git\n",
+    "%cd apps"
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
+    "### 3. Adding namespaces\n",
     "\n",
-    "#### Adding namespaces\n",
-    "For easier onboard to ArgoCD later on, we prefer to follow a name pattern for all our namespaces. Please use your team name as a prefix to the namespace name like so: `$TEAM_NAME-example-project`.\n",
+    "For easier onboarding to ArgoCD later on, we prefer to follow a name pattern for all our namespaces. Please use your team name as a prefix to the namespace name like so: `$TEAM_NAME-example-project`.\n",
     "#####  Base resources\n"
```

Comment:
Did you mean to use the level 4 header instead of level 5? Because this error causes myst.header and level 5 header makes it smaller text inside navbar than necessary.

Reply:
@mimotej good catch :+1: 
Updated